ZUGFeRD-Test/ZUGFeRD22Tests.cs

```diff
@@ -48,16 +48,19 @@ public void TestExtendedInvoiceWithIncludedItems()
             Stream s = File.Open(path, FileMode.Open);
             InvoiceDescriptor desc = InvoiceDescriptor.Load(s);
             s.Close();
-            
-            if(desc.TradeLineItems.Any())
+
+            desc.TradeLineItems.Clear();
+
+            TradeLineItem tradeLineItem = new TradeLineItem("line-001")
             {
-                desc.TradeLineItems[0].IncludedItems.Add(new IncludedItem
-                {
-                    Name = "Test",
-                    UnitQuantity = 1,
-                    UnitCode = QuantityCodes.C62
-                });
-            }
+                BilledQuantity = 10.0m, 
+                TaxPercent = 19.0m,
+                UnitCode = QuantityCodes.C62
+            };
+            desc.TradeLineItems.Add(tradeLineItem);
+
+            desc.TradeLineItems[0].AddIncludedItem("Test", 1, QuantityCodes.C62);
```

Comment:
could you use AddTradeLineItem() instead and use the instance that is returned from that call to invoke .AddIncludedItem()?

Could you also rename AddIncludedItem() to match the naming? I.e. AddIncludedReferenceProduct()